packages/db-schema/src/repositories/__tests__/boost.agentBoostTotals.integration.test.ts

```diff
@@ -0,0 +1,1114 @@
+import { and, eq } from "drizzle-orm";
```

Comment:
are these tests migrated from somewhere or are they brand new in this PR?

Reply:
Migrated from the `api` app.